src/JsonSchema/Validator.php

```diff
@@ -68,15 +76,34 @@ public function validate(&$value, $schema = null, $checkMode = null)
     }
 
     /**
-     * Alias to validate(), to maintain backwards-compatibility with the previous API
+     * Alias to validate(), to maintain backwards-compatibility with the previous API.
+     *
+     * Equivalent to calling validate() without setting any extra $checkMode options
+     *
+     * @api
+     *
+     * @param mixed $value  The value that should be validated against the schema - *not* a reference
+     * @param mixed $schema The schema to validate against
+     *
+     * @return int Bitwise list of error categories encountered during validation
```

Comment:
Maybe convey a little more information about the return, eg, tell us what value means no errors...